optimizely/config/polling_manager_test.go

```diff
@@ -74,3 +76,57 @@ func TestNewPollingProjectConfigManagerWithNull(t *testing.T) {
 	_, err := configManager.GetConfig()
 	assert.NotNil(t, err)
 }
+
+func TestNewPollingProjectConfigManagerWithSimilarDatafileRevisions(t *testing.T) {
+	mockDatafile1 := []byte(`{"revision":"42","projectId":"1000"}`)
+	mockDatafile2 := []byte(`{"revision":"42","projectId":"1001"}`)
```

Comment:
Ah, this is interesting. I'd recommend using the same `projectId` or better create 2 project configs from the same mock datafile. So you can just have 1 variable `mockDatafile` and create `projectConfig1` and `projectConfig2` from it.

Reply:
A slight change in both mock datafile's was required to verify that after calling `SyncConfig` using the second mock datafile, we still get project config generated for the first mock datafile. As per the suggestion, if we created 2 configs using the same mockDatafile, we won't be able to distinguish whether config for first datafile was returned or the second?

Reply:
My problem is with the premise of the test itself. It is an improbable event for project ID to be different. It does not line up with the mental model.

Why not pick a different field. Say set `botFiltering` to `False` in one datafile and to `True` in other datafile.

Reply:
Thanks, makes complete sense, i have made the suggested changes in the test-case.